### ISSUE TITLE
Is this a ethernet/ip scanner tool?

### ISSUE THREAD

**Issue by alexhjf** [ros-drivers/odva_ethernetip#9]
Hi,
Can this project be used to develop a simple Ethernet/IP scanner ?

**Comment by andreucm** [comment]
Yes, if you mean by "Ethernet/IP scanner" a process running on a machine which will manage (get/set data) an Ethernet/IP device such as a lidar, a motor drive or a safety switch. 
A good public instance,  is this omron lidar driver: https://github.com/ros-drivers/omron
best
andreu

**Comment by mean-ui-thread** [comment]
can it also make Ethernet/IP adapter devices? or it's just for doing Ethernet/IP scanner devices?

**Comment by jadamroth** [comment]
Hello all,

I know this is an old issue, but we've been working on a project for Ethernet/IP Scanner functionalities - https://github.com/nimbuscontrols/EIPScanner
